CoroutineTemplate/data/src/main/java/co/nimblehq/coroutine/data/response/Response.kt

```diff
@@ -0,0 +1,14 @@
+package co.nimblehq.coroutine.data.response
+
+import co.nimblehq.coroutine.domain.model.Model
+import com.squareup.moshi.Json
+
+data class Response(
+    @Json(name = "id") val id: Int?
+)
+
+private fun Response.toModel(): Model {
+    return Model(
+        id = this.id
+    )
+}
```

Comment:
I think the old implementation is practical when we need to come back to check the code style convention tho. Do you want to move all of the samples, including the code convention into the sample directory entirely? @Tuubz

Reply:
@lydiasama

> I think the old implementation is practical when we need to come back to check the code style convention

 Sorry, what do you mean by old implementation? 🙇  
 
 I think we could check the code style conventions from the sample code

Reply:
@Tuubz Do you think we should at least keep 1 API implementation to have a core structure in template for usecase and VM logic?

More samples for implementation will be checked out from sample project for sure but at least the core template initialization should have something to get started? 🤔